requirements.txt

```diff
@@ -1,11 +1,60 @@
-black == 23.3.0
-coverage == 7.2.2
-flake8 == 6.0.0
+aiosignal==1.3.1
+attrs==23.1.0
+black==23.3.0
+certifi==2023.7.22
+charset-normalizer==3.3.2
+click==8.1.7
+colorama==0.4.6
+contourpy==1.1.1
+coverage==7.2.2
+cycler==0.12.1
+distlib==0.3.7
+exceptiongroup==1.1.3
+filelock==3.13.1
+flake8==6.0.0
+fonttools==4.44.0
+frozenlist==1.4.0
+grpcio==1.59.2
+idna==3.4
+importlib-resources==6.1.1
+iniconfig==2.0.0
+joblib==1.3.2
+jsonschema==4.19.2
+jsonschema-specifications==2023.11.1
+kiwisolver==1.4.5
 matplotlib==3.6.3
+mccabe==0.7.0
+msgpack==1.0.7
+mypy-extensions==1.0.0
 numpy==1.24.1
+packaging==23.2
 pandas==1.4.1
+pathspec==0.11.2
+Pillow==10.1.0
+pkgutil_resolve_name==1.3.10
+platformdirs==3.11.0
+pluggy==1.3.0
+protobuf==4.25.0
+pycodestyle==2.10.0
+pyflakes==3.0.1
+pyparsing==3.1.1
 pytest==7.2.1
+python-dateutil==2.8.2
+pytz==2023.3.post1
+PyYAML==6.0.1
 ray==2.2.0
-scikit_learn==1.2.2
+referencing==0.31.0
+requests==2.31.0
+rpds-py==0.12.0
+scikit-learn==1.2.2
 scipy==1.10.1
-setuptools==44.0.0
```

Comment:
I don't think this should dissapear, it's used inside the [setup.py](https://github.com/taharallouche/Truth_Tracking-via-AV/blob/main/setup.py#L1) 

I'll try to run it locally with this requirements.txt and see if it breaks.
This local trial business is not optimal. A CI pipeline will be set to run before merging anything once [this MR](https://github.com/taharallouche/Truth_Tracking-via-AV/pull/32) is merged.